workflow-templates/static-analysis.yml

```diff
@@ -60,3 +60,6 @@ jobs:
 
       - name: Psalm
         uses: docker://vimeo/psalm-github-actions
+        # Uncomment this part in order to install dev dependencies before executing psalm
+        # with:
+        #  composer_require_dev: true
```

Comment:
As in https://github.com/doctrine/.github/pull/10, I don't understand the purpose of such templates.

Reply:
There are optional parts that some projects might not want to blindly copy paste. Using comments helpfully signals those.